### PR TITLE
metronome: bump

### PR DESCRIPTION
**Pull request by jkoelker**
## High-level description

What features does this change enable? What bugs does this change fix?

## Corresponding DC/OS tickets (required)

  - [D2IQ-ID](https://jira.d2iq.com/browse/D2IQ-ID) JIRA title / short description.

## Related tickets (optional)

  - [D2IQ-ID](https://jira.mesosphere.com/browse/D2IQ-<number>) JIRA title / short description.

### CODE REVIEW

**Comment by jkoelker** [comment]
@mesosphere-mergebot test all

**Comment by jkoelker** [comment]
@mesosphere-mergebot test

**Comment by jkoelker** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by jkoelker** [comment]
@mesosphere-mergebot backport 2.2 2.1

**Comment by jkoelker** [comment]
@mesosphere-mergebot backport 2.1.3

File: setup.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def get_advanced_templates():
         'Flask',
         'flask-compress',
         'urllib3==1.25.10',
-        'chardet',
+        'chardet<4.0',
```
Comment (jkoelker) [inline setup.py:62]: https://github.com/psf/requests/pull/5688 so until we bump requests, we need to use less than 4.

**Comment by jkoelker** [comment]
@mesosphere-mergebot test all

**Comment by jkoelker** [comment]
@mesosphere-mergebot test all

File: test-e2e/requirements.txt [hunk 2]
```diff
@@ -1,9 +1,9 @@
-git+https://github.com/dcos/dcos-e2e.git@2020.05.26.0
+git+https://github.com/dcos/dcos-e2e.git@2020.12.18.0
 attrs==19.1.0
 cryptography==3.2.1
 docker==4.0.2
 etcd3==0.11.1
-jwt==0.5.4
+PyJWT==1.7.1
```
Comment (jkoelker) [inline test-e2e/requirements.txt:6]: So historically it looks like when pip was using the legacy resolver, PyJWT was getting installed from a downstream dep somewhere *after* it had installed jwt here, so the effective pacakge of `import jwt` was PyJWT. Now that pip is using the new resolver, `import jwt` is getting jwt, which has a different api. :facepalm:

File: test-e2e/requirements.txt [hunk 3]
```diff
@@ -1,9 +1,9 @@
-git+https://github.com/dcos/dcos-e2e.git@2020.05.26.0
+git+https://github.com/dcos/dcos-e2e.git@2020.12.18.0
```
Comment (jkoelker) [inline test-e2e/requirements.txt:1]: With the new pip resolver, we needed a newer version of dcos-e2e which also supported `cryptography==3.2.1`

**Comment by jkoelker** [comment]
@mesosphere-mergebot request-review @vinodkone @timcharper

File: test-e2e/requirements.txt (hunk 2 above)
Comment (orsenthil) [reply test-e2e/requirements.txt:6]: That's an interesting find!

File: test-e2e/requirements.txt (hunk 3 above)
Comment (orsenthil) [reply test-e2e/requirements.txt:1]: Thanks for this update.

**Review by orsenthil — APPROVED** [review]
LGTM

**Review by vinodkone — APPROVED** [review]

**Pull request merged**